tests/common/helpers/ptf_tests_helper.py

```diff
@@ -113,19 +112,20 @@ def apply_dscp_cfg_teardown(duthost):
     reload_required = False
     for asic_id in duthost.get_frontend_asic_ids():
         swss = 'swss{}'.format(asic_id if asic_id is not None else '')
-        file_exists = duthost.shell("docker exec {} ls /usr/share/sonic/templates/ipinip.json.j2.tmp"
-                                    .format(swss))["rc"] == 0
-        if not file_exists:
+        try:
+            file_out = duthost.shell("docker exec {} ls /usr/share/sonic/templates/ipinip.json.j2.tmp".format(swss))
+        except Exception:
```

Comment:
Why do we need to ignore the exception? There shouldn't be exception if the docker is running.

Reply:
There is an ansible exception because the file cannot be found if not present, not a docker issue. When we ignore the exception, we are effectively saying the file cannot be found.